applications/portal/values.yaml

```diff
@@ -72,26 +72,39 @@ config:
   # -- Endpoint under `/api/` for the live TAP service on the instance, if present
   livetap: ""
 
+  # -- Each volume has a type, which is an object with keys "hostPath",
+  # "nfs", and "pvc".  At most one of these must have a non-null value.
+  # Each is an object.
+  # For the key "hostPath", the value is an object with a single string,
+  # "path", which indicates the hostPath to mount there.
+  # For the key "nfs", the value is an object with "path" and "server"
+  # keys, both strings.
+  # For the key "pvc", the value is an object with "size" and
+  # "storageClass" keys.  If "pvc" is chosen, the storage class must
+  # support mode "ReadWriteOnce" for the `privateWorkarea`,
+  # "ReadWriteMany"
+  # for `sharedWorkarea`, and "ReadOnlyMany" for `config`.
+  # If all objects have null values, an `emptyDir` is used for this volume.
+  # @default -- See `values.yaml`
```

Comment:
This turns into a really long documentation string and is going to cause problems for the page rendering. I think the best way to document something like this may be to add a documentation page to the Phalanx documentation under applications/portal about how to set up the storage and just link to it.

The constraints on mode should be moved to the documentation for each type of volume, since they're only relevant there and change for each type of volume.

Reply:
I've added the page and put both what I've done, and why, in there.  That will also be helpful, once merged, for my upcoming discussion with @gpdf about this.